### PR TITLE
Migration of themeMediaPLayer, viewSettings and notification to ES6 modules

### PR DESCRIPTION
**Pull request by Camc314**
**Issues**
part of #868

### CODE REVIEW

File: src/components/notifications/notifications.js [hunk 1]
```diff
@@ -1,182 +1,181 @@
...
+}
 
-            for (var i = 0, length = items.length ; i < length; i++) {
-                showNewItemNotification(items[i], apiClient);
-            }
-        });
-    }
+function getIconUrl(name) {
+    name = name || 'notificationicon.png';
+    return './components/notifications/' + name;
```
Comment (dmitrylyzo) [inline src/components/notifications/notifications.js:153]: Path should probably be relative to module itself, but I can't verify that - browser needs a secure connection (https?).

Comment (MrTimscampi) [reply src/components/notifications/notifications.js:153]: There is no other icon than `notificationicon.png` in that folder, so this is probably dead code?

Comment (Camc314) [reply src/components/notifications/notifications.js:153]: There is also `badge.png` in the folder as well.

Comment (dmitrylyzo) [reply src/components/notifications/notifications.js:153]: `require.toUrl('.')` returns `./components/notifications?v=12`.
Let's leave it as it is now.

**Pull request merged**